Mathlib/Topology/Category/TopCat/Limits/Konig.lean

```diff
@@ -36,139 +36,111 @@ discrete topology) in lemmas `nonempty_sections_of_finite_cofiltered_system` and
 -- Porting note: every ML3 decl has an uppercase letter
 set_option linter.uppercaseLean3 false
 
-open TopologicalSpace
-
 open CategoryTheory
 
 open CategoryTheory.Limits
 
-universe v u w
+universe u v w
 
 noncomputable section
 
 namespace TopCat
 
-variable {J : Type v} [SmallCategory J]
-
 section TopologicalKonig
 
-/-!
-## Topological Kőnig's lemma
-
-A topological version of Kőnig's lemma is that the inverse limit of nonempty compact Hausdorff
-spaces is nonempty.  (Note: this can be generalized further to inverse limits of nonempty compact
-T0 spaces, where all the maps are closed maps; see [Stone1979] --- however there is an erratum
-for Theorem 4 that the element in the inverse limit can have cofinally many components that are
-not closed points.)
-
-We give this in a more general form, which is that cofiltered limits
-of nonempty compact Hausdorff spaces are nonempty
-(`nonempty_limitCone_of_compact_t2_cofiltered_system`).
-
-This also applies to inverse limits, where `{J : Type u} [Preorder J] [IsDirected J (≤)]` and
-`F : Jᵒᵖ ⥤ Top`.
-
-The theorem is specialized to nonempty finite types (which are compact Hausdorff with the
-discrete topology) in lemmas `nonempty_sections_of_finite_cofiltered_system` and
-`nonempty_sections_of_finite_inverse_system` in the file `CategoryTheory.CofilteredSystem`.
-
-(See <https://stacks.math.columbia.edu/tag/086J> for the Set version.)
--/
```

Comment:
What's going on with this whole comment being deleted? I'm not sure where it would have gone if not this file.

Also, does this file need to be re-ported? It doesn't look like anything is really changed other than formatting, variable names, and explicit universe levels. (I haven't really been following how the porting process works, so this is a naive question.)

Reply:
Isn't this comment a duplicate of the module docstring?

Reply:
> It doesn't look like anything is really changed other than formatting, variable names, and explicit universe levels.

Part of the purpose of re-porting read to verify this; the original version was a somewhat-ad-hoc port as the sha didn't actually exist.

The universe variable changes are making CI fail, suggesting we didn't port things faithfully the first time (or leave a porting note to declare it deliberate)

Reply:
Oh, so it is; I didn't notice the "expand" button here.